src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -369,11 +371,11 @@ const SearchFieldList = ({ onSearch }: Props) => {
         <Grid container alignItems="center">
           <Accordion style={{ flexGrow: 1 }} elevation={3}>
             <AccordionSummary
-              expandIcon={<ExpandMore />}
+              expandIcon={<ExpandMore className={styles.people_expand_icon} />}
               id="more-search-options-header"
               aria-controls="more-search-options-controls"
             >
-              <Typography variant="h6" align="center">
+              <Typography variant="h6" align="center" className={styles.people_search_title}>
```

Comment:
This class doesn't exist in the imported stylesheet.

Reply:
see above comment - fixed